app/Controllers/SettingsController.php

```
@@ -208,6 +210,144 @@ public function updateEmailSettings(Request $request, Response $response, mysqli
+                    if ($update === false || $delete === false) {
+                        throw new \RuntimeException('Unable to prepare custom field statements');
+                    }
+                    foreach ($rows as $id => $row) {
+                        $id = (int) $id;
+                        if ($id <= 0 || !is_array($row)) {
+                            continue;
+                        }
+                        if (!empty($row['delete'])) {
+                            $delete->bind_param('i', $id);
+                            if (!$delete->execute()) {
+                                throw new \RuntimeException('Custom field delete failed: ' . $delete->error);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '293', column '43'). (undefined)

(MissingImport)